projects/canopy/src/lib/brand-icon/brand-icons.interface.ts

```diff
@@ -1,74 +1,137 @@
 // NOTE: this file is created automatically by `svg-to-ts`.
 // For more information check: https://github.com/kreuzerk/svg-to-ts.
 
-export const lgBrandIconCalendar: BrandIcon = {
+/* 🤖 this file was generated by svg-to-ts*/
```

Comment:
I'm glad they put this in. We could remove our own prompt at some stage.

Reply:
Yes, we could. Not sure if Paul is happy to do it in here, otherwise I'll do it later today.

Reply:
Mind if we leave it out of this one? Looking to get these icons into my project ASAP

Reply:
i'll change that later today